server/mediapublic/__init__.py

```diff
@@ -40,4 +43,13 @@ def main(global_config, **settings):
 
     config.include('velruse.providers.twitter')
 
+    config.add_view(mp_exc.handle_exceptions, context=Exception,
+                    permission=security.NO_PERMISSION_REQUIRED)
+    config.add_view(hooks.handle_exceptions,
```

Comment:
It could be nice down the line to also have 404's and 403's be JSON responses. The application  might be interested in handling them (especially 403's).